src/test/test-string-util.c

```diff
@@ -259,14 +259,19 @@ TEST(strextend_with_separator) {
 }
 
 TEST(strrep) {
-        _cleanup_free_ char *one = NULL, *three = NULL, *zero = NULL;
+        _cleanup_free_ char *one = NULL, *three = NULL;
+        char *onea, *threea;
         one = strrep("waldo", 1);
         three = strrep("waldo", 3);
-        zero = strrep("waldo", 0);
```

Comment:
Hmm, but why is this case removed? `strrep` still works fine with `0`, no?

Reply:
I'd like to make the test for two versions identical here. I can go further and also refuse the `n == 0` case for `strrep` if that's desirable 🤔

Reply:
Those calls do not have to be equivalent. `strrepa` is more limited, since it can only be used in scenarios where the inputs are well-controlled. So please keep `strrep` as it was and restore this one test.